README.md

```diff
@@ -63,6 +61,297 @@ The configuration YAML is validated against the rule schema before the scan is r
 the configuration are reported, and must be fixed before the scan can continue. Consult the
 documentation for each rule to see it's pattern filters and other configurable properties.
 
+## Finding hash
+
+TODO: This section is WIP, as the hash algorithm is being upgraded.
+
+To enable de-duplication of findings, a hash is calculated for each finding. The hash is the
+`sha256` digest of a canonical content string for the finding.
+
+The digest is then the `sha256` of the content string, in hex.
```

Comment:
It's not clear what is this `content string`? I'd assume the content contains all the properties listed below, but would be good to mention this explicitly.